src/Routes/Organigram.elm

```diff
@@ -14,7 +14,114 @@ organigram model =
     in
         div [ class "w-60-ns center" ]
             [ h1 [ class "tc f1" ] [ text ("Structure of " ++ (displayOrganisation org) ++ " team") ]
-            , p [ class "f3 w60 mh1 tc" ] [ text "hello" ]
+            , div [ class "content" ]
+                [ figure [ class "org-chart cf" ]
+                    [ ul [ class "administration" ]
+                        [ li []
```

Comment:
When you see lots of clear repetition like there is here, it's 100% worth making that info functions. eg:

you could have something like:

```elm
item : String -> Html msg
item name =
    li []
        [ a [ href "#" ]
            [ span []
                [ text name ]
            ]
        ]
```

and instead of:

```elm
ul [ class "subdirector" ]
    [ li []
        [ a [ href "#" ]
            [ span []
                [ text "Assistante Director" ]
            ]
        ]
    ]
```

it would be:

```elm
ul [ class "subdirector" ]
    [ item "Assistante Director" ]
    ]
```

There's a lot more you can do like this in this file that would make it much cleaner, I'd recommend playing around with it at some point to get a feel for abstracting out pieces of html.